docs/.vuepress/components/Disqus.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div id="disqus_thread"></div>
+</template>
+<script>
+export default {
+  mounted() {
+    var disqus_config = function() {
+      this.page.url = window.location.origin;
```

Comment:
Et je sais pas si c'est voulu, mais `this.page` est rattaché à ta fonction et non pas a ton composant

Reply:
Yep c'est voulu